<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWindow</class>
 <widget class="QWidget" name="HelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справка</string>
  </property>
  <property name="windowIcon">
   <iconset resource="menu.qrc">
    <normaloff>:/menu/src/Menu/On.png</normaloff>:/menu/src/Menu/On.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>661</width>
     <height>911</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PresentScript</family>
     <pointsize>24</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #696969;
color: #FFFFFF;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>940</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kashima RUS by Cop</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
color: black;</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>940</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kashima RUS by Cop</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
color: black;</string>
   </property>
   <property name="text">
    <string> На главную </string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>940</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kashima RUS by Cop</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
color: black;</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="menu.qrc"/>
 </resources>
 <connections/>
</ui>
